angular-spa-ui/js/common/configService-srv.js

```diff
@@ -1,46 +1,42 @@
 module.exports = function ($injector) {
 
-    var getConfig = function (configName) {
+    return {
+        getData: function (configName, keysPath) {
 
-        try {
-            return angular.copy($injector.get(configName));
-        } catch (err) {
-            throw Error('Can not get config ' + configName);
-        }
+            var config = this.getConfig(configName),
```

Comment:
Variable declarations should be the same var a, b, c; or var a; var b; var c around all application